### PR TITLE
Feature/fix bugs

### PR DESCRIPTION
**Pull request by swaroopar**
fixes https://github.com/eclipse-xpanse/xpanse/issues/1846

### CODE REVIEW

File: samples/develop/HuaweiCloud-Compute-opentofu-dev.yml [hunk 1]
```diff
@@ -422,11 +422,6 @@ deployment:
       override_special = "#%@"
     }
 
-    resource "huaweicloud_kps_keypair" "keypair" {
```
Comment (baixinsui) [inline samples/develop/HuaweiCloud-Compute-opentofu-dev.yml:425]: The `admin_pass` configuration will not take effect when using keypair? Let me test on the FlexibleEngine

Comment (swaroopar) [reply samples/develop/HuaweiCloud-Compute-opentofu-dev.yml:425]: @baixinsui yes correct.

**Comment by baixinsui** [comment]
LGTM

**Review by iskey — APPROVED** [review]
LGTM

**Pull request merged**